test-resources/odata-service-specs-e2e/v4/API_TEST_SRV/API-TEST_SRV.edmx

```diff
@@ -6,14 +6,21 @@
                 <EntitySet Name="TestEntity" EntityType="TestService.TestEntity">
                     <NavigationPropertyBinding Path="ToMultiLink" Target="TestEntityLink"/>
                 </EntitySet>
+                <EntitySet Name="TestEntityWithMultipleKeys" EntityType="TestService.TestEntityWithMultipleKeys"/>
```

Comment:
[q] I assume this is not a manual update and is retrieved from the cap app.

Reply:
I'm not aware that we have a workflow to automatically re-generate those files. I've added the command to regenerate the edmx to the cap app [here](https://github.com/SAP/cloud-sdk-js/blob/bfa038ea5f907d0494afb49393bdecd0e19e9001/test-packages/e2e-tests/package.json#L24), but applying those changes here is a manual effort. I'd be happy to see this improved, but my understanding is that fully automating this is not trivial.